src/Homie/Boot/BootConfig.cpp

```diff
@@ -175,21 +176,22 @@ void BootConfig::_onWifiStatusRequest(AsyncWebServerRequest *request) {
 
   json["status"] = status;
   String output;
-  json.printTo(output);
+  serializeJson(generatedJsonDoc, output);
 
   request->send(200, FPSTR(PROGMEM_CONFIG_APPLICATION_JSON), output);
 }
 
 void BootConfig::_onProxyControlRequest(AsyncWebServerRequest *request) {
   Interface::get().getLogger() << F("Received proxy control request") << endl;
-  DynamicJsonBuffer parseJsonBuffer(JSON_OBJECT_SIZE(1));
-  const char* body = (const char*)(request->_tempObject);
-  JsonObject& parsedJson = parseJsonBuffer.parseObject(body);  // do not use plain String, else fails
-  if (!parsedJson.success()) {
+  StaticJsonDocument<JSON_OBJECT_SIZE(1)> parseJsonDoc;
+  char* body = (char*)(request->_tempObject);
+  if (deserializeJson(parseJsonDoc, body) != DeserializationError::Ok || !parseJsonDoc.is<JsonObject>()) { // do not use plain String, else fails
```

Comment:
Does the comment still make any sense? The comment also doesn't describe why. I would be in favor to remove this comment.

Reply:
I have no idea why it was there the first place. I'll remove it.